user-interface/src/main/java/life/qbic/datamanager/views/projects/project/measurements/registration/MeasurementUpload.java

```diff
@@ -113,13 +113,21 @@ public MeasurementUpload(
     upload.addFileRejectedListener(this::onFileRejected);
     upload.addFailedListener(this::onUploadFailed);
     upload.addFileRemovedListener(this::onFileRemoved);
-    this.uploadItemsDisplay = new UploadItemsDisplay(upload);
+    this.uploadedItemsDisplay = new UploadItemsDisplay(upload);
```

Comment:
```suggestion
    this.uploadedItemsDisplay = new UploadedItemsDisplay(upload);
```
Should we rename the class to stay consistent? What were your thought in changing `uploadItemsDisplay` to `uploadedItemsDisplay`? Do they apply to the class as well?